newbank/server/NewBank.java

```diff
@@ -178,13 +178,13 @@ private String payMoney(CustomerID customer, double amount, String source, Strin
 					return "FAIL: Insufficient funds in source account";
 				} else {
 					double currentBalance = customerAccount.getCurrentBalance();
-					double newBalance = currentBalance - amount;
+					double newBalance = currentBalance;
 					customerAccount.setCurrentBalance(newBalance);
-					customerAccount.addMovement(new AccountMovement(source, payee, -amount, LocalDateTime.now()));

```

Comment:
@abidemiraji I'm sure about this change (line 183). The idea of the minus sign is to make a negative movement. I guess, in general, that all movements can be positive but either debit or credit, but the way I implemented it, it is the sign of the value that tell what type of movement it is.